kubernetes/samples/scripts/domain-lifecycle/startDomain.sh

```diff
@@ -0,0 +1,78 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a deployed WebLogic domain by patching 'spec.serverStartPolicy'
+  attribute of the domain resource to 'IF_NEEDED'. This change will cause the operator
+  to initiate startup of domain's WebLogic server instance pods if the pods are not
+  already running.
+ 
+  Usage:
+ 
+    $(basename $0) [-n mynamespace] [-d mydomainuid] [-m kubecli]
+  
+    -d <domain_uid>     : Domain unique-id. Default is 'sample-domain1'.
+
+    -n <namespace>      : Domain namespace. Default is 'sample-domain1-ns'.
+
+    -m <kubernetes_cli> : Kubernetes command line interface. Default is 'kubectl' if KUBERNETES_CLI env
+                          variable is not set. Otherwise default is the value of KUBERNETES_CLI env variable.
+
+    -h                  : This help.
+   
+EOF
+exit $1
+}
+
+kubernetesCli=${KUBERNETES_CLI:-kubectl}
+domainUid="sample-domain1"
+domainNamespace="sample-domain1-ns"
+
+while getopts "n:d:m:h" opt; do
+  case $opt in
+    n) domainNamespace="${OPTARG}"
+    ;;
+    d) domainUid="${OPTARG}"
+    ;;
+    m) kubernetesCli="${OPTARG}"
+    ;;
+    h) usage 0
+    ;;
+    *) usage 1
+    ;;
+  esac
+done
+
+
+set -eu
+set -o pipefail
+
+if ! [ -x "$(command -v ${kubernetesCli})" ]; then
+  fail "${kubernetesCli} is not installed"
```

Comment:
This should use the existing helper functions that does same (see startCluster.sh).